lms/templates/courseware/courseware.html

```diff
@@ -234,6 +234,25 @@
     </div>
 % endif
     <section class="course-content" id="course-content">
```

Comment:
I am still curious to know how UNDEFINED works. Can you please explain it to me?